api/controllers/user.js

```diff
@@ -16,10 +15,6 @@ module.exports = {
   },
 
   me: (req, res) => {
-    User.findByPk(req.user.id)
-      .then(model => authorizer.me(req.user, model))
-      .then(currentUser => userSerializer.serialize(currentUser))
-      .then(userJSON => res.json(userJSON))
-      .catch(err => res.error(err));
+    res.json(userSerializer.toJSON(req.user));
```

Comment:
@davemcorwin -  how does this now handle errors? is it the same in the end as the catch block?

Reply:
Added generic error handler that will handle this. Above, call next with any errors to forward there.  See https://expressjs.com/en/guide/error-handling.html.